### PR TITLE
Andy homepage redesign

### PR DESCRIPTION
**Pull request by andylouisqin**
STILL NEED TO BE COMPLETED:
* Adding the Active and Past Projects section above the animations
* Hooking up Aleks' modal to the "Donate Now" button at the bottom
* Hooking up the links in the nav bar to actual pages

Screenshots 4 dayz.
screen shot 2015-03-15 at 9 54 02 pm
screen shot 2015-03-15 at 9 54 12 pm
screen shot 2015-03-15 at 9 54 21 pm
screen shot 2015-03-15 at 9 54 31 pm
screen shot 2015-03-15 at 9 54 36 pm

### CODE REVIEW

**Comment by andylouisqin** [comment]
I may also have messed up the mobile responsiveness entirely lol. Need to learn how to Foundation.

**Comment by noahsark769** [comment]
Nice!

Two primary concerns:
1. Master needs to merged in haha
2. Is the top bar still Django-CMSable? It needs to be, so you should talk to @vivekraghuram if you have questions about django-cms

File: revolv/static/footer.scss [hunk 1]
```diff
@@ -49,6 +44,11 @@ $footer-divider-height: 2px;
     }
 }
 
+#footer {
```
Comment (noahsark769) [inline revolv/static/footer.scss]: Wait so now we have .footer AND #footer?

File: revolv/static/home.scss [hunk 2]
```diff
@@ -57,51 +71,186 @@
     display: block;
     width: 100%;
     height: $revolv-cover-photo-height;
+    // min-height: 100%;
```
Comment (noahsark769) [inline revolv/static/home.scss]: Can just delete :)

File: revolv/static/footer.scss (hunk 1 above)
Comment (andylouisqin) [inline revolv/static/footer.scss]: That's embarrasing

**Comment by andylouisqin** [comment]
Thanks!

1. What does this entail?
2. Yep!
screen shot 2015-03-16 at 1 13 43 am

**Comment by chenhoward** [comment]
@andylouisqin ++ Dude what a beast.  You actually put images in your pr comment so much determination.

**Comment by noahsark769** [comment]
Aw yeah revolv++

**Comment by noahsark769** [comment]
Wait @andylouisqin did you push your merge changes to this branch? It says it still hasn't been updated

**Comment by andylouisqin** [comment]
Not yet! Will look to do sometime tonight



—

File: revolv/static/nav.scss [hunk 3]
```diff
@@ -37,6 +37,12 @@ nav.header {
     .user-controls > .dropdown a.js-generated {
         display: none;
     }
+
+    /* Overriding Foundation colors */
```
Comment (noahsark769) [inline revolv/static/nav.scss]: This seems a bit much: I guess we probably need all these selectors due to CSS specificity right? Can we at least have a comment explaining what this is doing specifically/which element it's actually styling?

File: revolv/templates/base/home.html [hunk 4]
```diff
@@ -32,71 +32,196 @@
...
+        <img class="person" src="{% static "images/revolv-man.svg" %}" />
+        <img 
+            data-615="margin-bottom: -84px;"
+            data-732="margin-bottom: -670px;"
+            data-733="opacity[begin]: 1;"
+            data-734="opacity[end]: 0;"
+            class="coin" src="{% static "images/coin.svg" %}" />
+        <img class="person" src="{% static "images/revolv-woman.svg" %}" />
+
+        <svg role="img" title="factory" class="factory">
```
Comment (noahsark769) [inline revolv/templates/base/home.html]: We definitely need to put this svg content into a separate file

File: revolv/templates/base/partials/nav.html [hunk 5]
```diff
@@ -21,7 +21,21 @@
                {% show_menu 0 100 100 100 "base/partials/menu.html" %}
            </ul>
 
-           <ul class="right">
+           <ul class="right header-small-text">
+
+               <li>
```
Comment (noahsark769) [inline revolv/templates/base/partials/nav.html]: Everything here except the signin/signout will be handled by django-cms, so we shouldn't hardcode it for now

**Comment by noahsark769** [comment]
Alright, some things:

1. For some reason, the dashboard and signout links don't appear if you're (1) signed in and (2) scrolled all the way to the top of the page and you (3) mouse over the "administrator"/"ambassador" nav bar item. Not sure why this is, but it's a bug :(
2. For my 13 inch screen, the text almost is hidden by the bottom of the page. Think we could make the space between the spine items less?
screen shot 2015-03-21 at 10 54 41 am
3. For tablet screens, it goes off the bottom :( 
screen shot 2015-03-21 at 10 57 20 am
4. On tablet and phones, the header bar is pretty cramped and not scrollable...do you think there's anything we could do about that? 
screen shot 2015-03-21 at 10 57 36 am
screen shot 2015-03-21 at 10 58 08 am
5. On tablet portrait the text actually overflows the graphics haha. I think the solution here is to make the spine text smaller and move the graphics over to the side - won't be centered anymore, but will at least be readable 
screen shot 2015-03-21 at 10 58 55 am
6. Fonts are a bit large on mobile haha: 
screen shot 2015-03-21 at 10 59 07 am
screen shot 2015-03-21 at 11 00 51 am
7. I actually don't know about that infographic on mobile, it's so small :( Maybe we could have the spine items, not animated, on the mobile page, then have a link to /about/ to learn more? That's what we do on the current home page. 
screen shot 2015-03-21 at 11 02 19 am

Most of these will be not too hard to fix I think, and on the plus side the code looks really good! Hopefully we can get this merged soon so you can move on to more backend tasks haha

**Comment by andylouisqin** [comment]
Thanks for all the feedback @noahsark769! I'll get to it today

**Comment by andylouisqin** [comment]
New commits! My solutions to the issues:
* Nav bar bug: didn't use Foundation's sticky nav, has weird behavior
* As a result, every page needs to now have a top margin as wide as the (60px) nav bar
* Image too small on mobile: Made animations work for mobile, check 'em out! Can't wait to try on my actual phone

**Comment by noahsark769** [comment]
Wow, this looks awesome! Just a couple of more things:

1. Numbers 2 and 3 of the mobile version of the "nav spine" don't become visible until you scroll past them on landscape mobile
screen shot 2015-03-25 at 5 50 05 pm
2. On portrait mobile, can you just add a little bit more margin to the title? It looks very cramped right now: 
screen shot 2015-03-25 at 5 54 41 pm

After those fixes I think this should finally be good to merge!

**Comment by noahsark769** [comment]
OKAY ONE MORE THING I JUST SAW: since the background color has now changed, the auth pages need a style tweek for their headings in order to match the backround: you should just have to change the background color in the call to `text-with-divider` I think
screen shot 2015-03-25 at 7 41 11 pm

**Comment by noahsark769** [comment]
@andylouisqin I think the spacer should go in nav.html, right?

**Comment by andylouisqin** [comment]
I'd have that, but then the landing page would be spaced down when I don't want it. Maybe I can make a CSS exception for that page alone



—

**Comment by noahsark769** [comment]
What you might be able to do is have a `.spacer` class which has a set width in the nav.scss file, but have an override in `home.scss`?

**Comment by noahsark769** [comment]
ITS TIME TO MERGE THIS FINALLY

**Pull request merged**